src/gui/tray/activitylistmodel.h

```diff
@@ -55,6 +55,7 @@ class ActivityListModel : public QAbstractListModel
         DisplayPathRole,
         PathRole,
         AbsolutePathRole,
+        DisplayLocationRole, // Not the path to the file but its folder
```

Comment:
I am not sure I understand the comment
is this about showing only the parent folder name or something else ?

Reply:
Yes, this is used in the tooltip to show the display (i.e. relative to the Nextcloud root folder) path to the file's/folder's parent folder. I can clarify the comment

Reply:
Reworded it, should hopefully be a bit clearer